tox.ini

```diff
@@ -33,6 +33,7 @@ passenv =
 extras =
     mlflow
     tensorboard
+    scatter
```

Comment:
The package is named `torch_scatter`,  cf. https://github.com/rusty1s/pytorch_scatter

Reply:
Correct, but the extra in `setup.cfg` is just called scatter. This is not the reason why it's not working 😢